src/rewriter/rewrite_db.h

```diff
@@ -60,6 +73,46 @@ class RewriteDb
                Node cond,
                Node context,
                bool isFlatForm = false);
```

Comment:
can you clarify what being in flat form means?

Reply:
It's a deprecated feature, I'll remove.